index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/Styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  </head>
+  <body>
+    <section class="navigation">
+      <h3>Byiringiro Moise</h3>
+      <span class="material-symbols-outlined"> menu </span>
+    </section>
+    <section class="profile">
+        <div class="introduction">
+<h1>Hey There. I'm Moise</h1>
+<h4>I'am a Software Developer</h4>
+<p>
+    I can help you build a product, feature or website look through some of my work and experience!
+    if you like what you see and have a project you need coded, don't hesitate to contact me.
+</p>
+        </div>
+        <div class="social-media-icons">
+<a href=""><img src="/items/Github-icon.png" alt="github icon"></a>
+<a href=""><img src="/items/linkedin-icon.png" alt=""></a>
+<a href=""><img src="/items/Icon - Angelist.png" alt=""></a>
+<a href=""><img src="/items/Icon - Twitter.png" alt=""></a>
+<a href=""><img src="/items/M-icon.png" alt=""></a>
+
```

Comment:
- [ ] I think it will be better if your list of icons is wrapped in `ul` tag to group them since the icons are a list, and also wrap each icon in  a `li` tag with each icon being a link (`a` tag)

e.g. 

 ```
<ul class="social-media">
        <li id="angellist">
            <a href="">
                <img src="images/angellist.png" alt="angellist">
            </a>
        </li>
         <li id="medium">
            <a href="">
                <img src="images/medium.png" alt="medium">
            </a>
        </li>
       ...
    </ul>
```